internal/configs/resource.go

```diff
@@ -756,6 +785,39 @@ func (r *ProviderConfigRef) String() string {
 	return r.Name
 }
 
+func (r *ProviderConfigRef) InstanceValidation(subject string, isInstanced bool) hcl.Diagnostics {
```

Comment:
```suggestion
func (r *ProviderConfigRef) InstanceValidation(blockType string, isInstanced bool) hcl.Diagnostics {
```

This is not _super_ important, but I found myself checking back to the caller to find out what exactly "subject" meant here, so I figured it was worth choosing a more specific name to benefit a future reader.

Of course, if you accept this suggestion then the definition of `summary` below would also need to be updated to use the new variable name.

Reply:
Fixed